src/count/bower.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
As long as these dependencies and ignored files are in the main bower.json for o-comments this file can probably be removed

Reply:
awesome, will do 😄